pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -1629,6 +1629,26 @@ func validateDisks(field *k8sfield.Path, disks []v1.Disk) []metav1.StatusCause {
 			})
 		}
 
+		ioModeOK := false
+		switch disk.IO {
+		case "":
+			ioModeOK = true
+		case v1.IODefault:
+			ioModeOK = true
+		case v1.IONative:
+			ioModeOK = true
+		case v1.IOThreads:
+			ioModeOK = true
+		}
+		if !ioModeOK {
```

Comment:
Perhaps it's better to replace this switch with a simple if?
if disk.IO != "" && disk.IO != v1.IODefault && disk.IO != v1.IONative && disk.IO != v1.IOThreads {

What do you think?

Reply:
I didn't use `if` condition because it makes it less readable IMO but I don't have a strong opinion on it...

Reply:
specifically if we will add more conditions (and we will add `io_uring` in the future for example)

Reply:
I'm somewhat ambivalent here, but I'd point out that we're modifying a bool, so the outcome is either "ok" or "not ok", thus when we add `io_uring` or any other mode, an if is still reasonable. for readability you could always break it up across lines:

```
if disk.IO != "" && \
    disk.IO != v1.IODefault && \
    disk.IO != v1.IONative && \
    disk.IO != v1.IOThreads
```